flow-report/test/summary/category-test.tsx

```diff
@@ -8,14 +8,14 @@ import {FunctionComponent} from 'preact';
 import {render} from '@testing-library/preact';
 
 import {SummaryTooltip} from '../../src/summary/category';
-import {Util} from '../../../report/renderer/util';
 import {flowResult} from '../sample-flow';
 import {I18nProvider} from '../../src/i18n/i18n';
 import {FlowResultContext} from '../../src/util';
 
 let wrapper: FunctionComponent;
 
 beforeEach(() => {
+  // Include sample flowResult for locale in I18nProvider.
```

Comment:
Drive by change to this test so the expectations don't change every time the sample is updated.